addons/hr_expense/models/hr_expense.py

```diff
@@ -59,17 +59,10 @@ def _default_employee_id(self):
     product_description = fields.Html(compute='_compute_product_description')
     product_uom_id = fields.Many2one(
         comodel_name='uom.uom',
-        string="Unit of Measure",
+        string="Unit",
         compute='_compute_uom_id', precompute=True, store=True,
```

Comment:
Maybe you'll need to keep a domain here based on the `allowed_uom_ids` for that product?